staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go

```diff
@@ -19,25 +19,23 @@ package expose
 import (
 	"fmt"
 	"regexp"
-	"strconv"
 	"strings"
 
 	"github.com/spf13/cobra"
 	"k8s.io/klog/v2"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructuredscheme"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/printers"
 	"k8s.io/cli-runtime/pkg/resource"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"k8s.io/kubectl/pkg/generate"
+	"k8s.io/kubectl/pkg/generate/versioned"
```

Comment:
We don't want to use generators, we want to reduce and/or eliminate their usage.

In fact, this same import was removed in 1.24:
https://github.com/kubernetes/kubernetes/pull/106824

Reply:
Thank you, I will modify it